package.json

```diff
@@ -37,7 +37,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.8",
+    "react": ">=0.14.8 <15.2.0",
```

Comment:
No worries I only did this because I couldn't guarantee compatibility after
15.1 I'll make a new pull request.

On 27 Jul 2016 9:49 p.m., "Alan Clarke" <notifications@github.com> wrote:

> In package.json
> <https://github.com/alanclarke/time-input/pull/12#discussion_r72520719>:
>
> > @@ -37,7 +37,7 @@
> >      "karma-webpack": "^1.7.0",
> >      "mocha": "^2.4.5",
> >      "raw-loader": "^0.5.1",
> > -    "react": "^0.14.8",
> > +    "react": ">=0.14.8 <15.2.0",
>
> so, the reason ^0.14.0 || ^15.0.0 is better is because ^ lets minor and
> major updates through whereas <15.2.0 would mean that 15.2.1 wouldn't be
> allowed even though the .1 is only a bugfix patch update
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alanclarke/time-input/pull/12/files/ffb249310cf3000a9cad2706191751afe600b091#r72520719>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFGXftDctid_SpbTJNGT6cO6GS8S4QTjks5qZ8RsgaJpZM4JWBfU>
> .
>